apps/remote_control/lib/lexical/remote_control/mix.ex

```diff
@@ -11,13 +10,17 @@ defmodule Lexical.RemoteControl.Mix do
     end
   end
 
+  def with_lock(fun) do
+    RemoteControl.with_lock(__MODULE__, fun)
+  end
+
   def in_project(%Project{} = project, fun) do
     # Locking on the build make sure we don't get a conflict on the mix.exs being
     # already defined
 
     old_cwd = File.cwd!()
 
-    Build.with_lock(fn ->
+    with_lock(fn ->
```

Comment:
If it's not used elsewhere, my preference would be to not introduce a new function, and instead inline this:

```suggestion
    RemoteControl.with_lock(__MODULE__, fn ->
```

Reply:
I'll make this private.